src/elm/App/Update.elm

```diff
@@ -21,6 +21,10 @@ type Action
   -- NoOp actions
   | NoOp
 
+dragActions : Signal Action
+dragActions =
+    Signal.map ChildTimelineAction Timeline.Update.startTimeActions
```

Comment:
yeah, I didn't find a way to convert shape to Element. We'll have to dig in a bit..